comp/vision/tag_finder24.py

```diff
@@ -191,32 +191,32 @@ def __init__(self, serial, camList):
         self.at_detector.addFamily("tag36h11")
 
     def analyze(self, request, camera):
-        potentialTags = self.estimatedTagPose.get()
-        potentialArray = []
-        z = []
-        for Blip24s in potentialTags:
-            translation = Blip24s.pose.translation()
-            if translation.Z() < 0:
-                continue
-            rvec = np.zeros((3, 1), np.float32)
-            tvec = np.zeros((3, 1), np.float32)
-            object_points = np.array(
-                [translation.X(), translation.Y(), translation.Z()], np.float32
-            )
-            (point2D, _) = cv2.projectPoints(
-                object_points, rvec, tvec, camera.mtx, camera.dist
-            )
-            if (
-                point2D[0][0][0] > 0
-                and point2D[0][0][0] < 1456
-                and point2D[0][0][1] > 0
-                and point2D[0][0][1] < 1088
-            ):
+        # potentialTags = self.estimatedTagPose.get()
```

Comment:
big swaths of commented-out code are bad.  maybe you could make a copy of tag_finder24 with this stuff in it?